cli/assign_node_identity.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package cli
+
+import (
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
+	acpIdentity "github.com/sourcenetwork/defradb/acp/identity"
+	"github.com/sourcenetwork/immutable"
+	"github.com/spf13/cobra"
+)
+
+func MakeAssignNodeIdentityCommand() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "assign-node-identity [identity]",
+		Short: "Assign an identity to the node",
+		Long: `Assign an identity to the node.
+
+Identity is hex-formatted private key.
+Node uses the identity to be able to exchange encryption keys with other nodes.
+		
+Example to assign an identity to the node:
+  defradb client assign-node-identity 028d53f37a19afb9a0dbc5b4be30c65731479ee8cfa0c9bc8f8bf198cc3c075f 
```

Comment:
suggestion:

Why not, for this do: `defradb client node-identity assign 028d53f37a`

And for the other one do: ``defradb client node-identity get``

Reply:
I like that